i18n/ja.toml

```diff
@@ -28,6 +28,9 @@ other = "Twitter"
 [community_youtube_name]
 other = "YouTube"
 
+[deprecation_title]
+other = "現在表示しているのは、次のバージョン向けのドキュメントです。Kubernetesバージョン:"
```

Comment:
@shuuji3  , Changes look good.
I would list `deprecation_file_warning` before `deprecation_warning`, but this is okay to ignore for this change.

Reply:
Thanks for pointing out. Yes, it's preferable to sort by alphabetical order. I'll fix the order.